esphome/components/waveshare_epaper/display.py

```diff
@@ -98,6 +101,9 @@
     "WaveshareEPaper13P3InK", WaveshareEPaper
 )
 GDEW0154M09 = waveshare_epaper_ns.class_("GDEW0154M09", WaveshareEPaper)
+WaveshareEPaper7In5BV2 = waveshare_epaper_ns.class_(
+    "WaveshareEPaper7In5BV2", WaveshareEPaperPolled
+)
```

Comment:
Please follow the naming scheme that is already used
```suggestion
WaveshareEPaper7P5InBV2RB = waveshare_epaper_ns.class_(
    "WaveshareEPaper7P5InBV2RB", WaveshareEPaperPolled
)
```